<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GoodsSetPage</class>
 <widget class="QWidget" name="GoodsSetPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>501</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="skinWidget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#skinWidget{
	background-color:#ffffff;
}</string>
     </property>
     <widget class="CategoryWidget" name="categoryWidget" native="true">
      <property name="geometry">
       <rect>
        <x>82</x>
        <y>11</y>
        <width>406</width>
        <height>111</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>406</width>
        <height>111</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="categorySetLabel">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>11</y>
        <width>65</width>
        <height>19</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>分类设置：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
      </property>
     </widget>
     <widget class="QLabel" name="goodsSetLabel">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>130</y>
        <width>65</width>
        <height>19</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>商品设置：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
      </property>
     </widget>
     <widget class="GoodsWidget" name="goodsWidget" native="true">
      <property name="geometry">
       <rect>
        <x>82</x>
        <y>130</y>
        <width>406</width>
        <height>360</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>406</width>
        <height>360</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="goodsNextButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>250</y>
        <width>55</width>
        <height>55</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>55</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>55</width>
        <height>55</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>55</width>
        <height>55</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border:none;
	font-size:12px;
	color:#ffffff;
	padding-top:20px;
	border-image: url(:/saleImage/goods_down.png);
}

QPushButton:pressed{
	border-image: url(:/saleImage/goods_down_press.png);
}</string>
      </property>
      <property name="text">
       <string>1/12页</string>
      </property>
     </widget>
     <widget class="QPushButton" name="goodsFrontButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>189</y>
        <width>55</width>
        <height>55</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>55</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>55</width>
        <height>55</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>55</width>
        <height>55</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border:none;
	border-image: url(:/saleImage/goods_up.png);
}

QPushButton:pressed{
	border-image: url(:/saleImage/goods_up_press.png);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="categoryNextButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>128</y>
        <width>55</width>
        <height>55</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>55</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>55</width>
        <height>55</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>55</width>
        <height>55</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border:none;
	font-size:12px;
	color:#ffffff;
	padding-top:20px;
	border-image: url(:/saleImage/category_down.png);
}

QPushButton:pressed{
	border-image: url(:/saleImage/category_down_press.png);
}</string>
      </property>
      <property name="text">
       <string>1/12页</string>
      </property>
     </widget>
     <widget class="QPushButton" name="categoryFrontButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>70</y>
        <width>55</width>
        <height>55</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>55</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>55</width>
        <height>55</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>55</width>
        <height>55</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border:none;
	border-image: url(:/saleImage/category_up.png);
}

QPushButton:pressed{
	
	border-image: url(:/saleImage/category_up_press.png);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="newCategoryButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>10</y>
        <width>55</width>
        <height>55</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>55</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>55</width>
        <height>55</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>55</width>
        <height>55</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#newCategoryButton {
	color: rgb(255, 255, 255);
	border-image: url(:/setImage/add_category_normal.png);
}

QPushButton#newCategoryButton:pressed {
	border-image: url(:/setImage/add_category_press.png);
}</string>
      </property>
      <property name="text">
       <string>添加
分类</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GoodsWidget</class>
   <extends>QWidget</extends>
   <header>GoodsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CategoryWidget</class>
   <extends>QWidget</extends>
   <header>CategoryWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
